h/views/organizations.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from pyramid.view import view_config
+
+
+@view_config(route_name='organization_logo', request_method='GET', renderer='svg', http_cache=600)
+def organization_logo(logo, request):
+    return logo
```

Comment:
All the work has been done in https://github.com/hypothesis/h/pull/4902 and <https://github.com/hypothesis/h/pull/4903>, so all the actual view needs to do is return its argument.

The `http_cache=600` causes browsers to cache the SVG files for 10 mins.